DevSkim-DotNet/Microsoft.DevSkim.CLI/Commands/SuppressionCommand.cs

```diff
@@ -74,7 +81,10 @@ public int Run()
                         Console.Error.WriteLine($"{potentialPath} specified in sarif does not appear to exist on disk.");
                     }
 
-                    var theContent = File.ReadAllText(potentialPath).Split(Environment.NewLine);
+                    var theContent = File.ReadAllText(potentialPath).Split(
```

Comment:
I think it would be simpler to just use ReadAllLines here:

https://learn.microsoft.com/en-us/dotnet/api/system.io.file.readalllines?view=net-7.0